tasmota/support_features.ino

```diff
@@ -787,8 +787,9 @@ void ResponseAppendFeatures(void)
 #ifdef USE_SHIFT595
     feature8 |= 0x00080000;  // xdrv_60_shift595.ino
 #endif
-
-//    feature8 |= 0x00100000;
+#if defined(USE_I2C) && defined(USE_DS3231)
```

Comment:
Duplicate. Is already in this file. Pls remove.

Next time use a global search within VSC to track any name.

Reply:
Done.

Reply:
If this will get approved, I'll start working on documentation.